prep-javascript-functions/main.js

```diff
@@ -0,0 +1,71 @@
+function addTwoNumbersTogether(x, y) {
+  return x + y;
+}
+
+var sum = addTwoNumbersTogether(4, 4);
+
+console.log('sum', sum);
+
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+
+var addTwoNumbersResult = addTwoNumbers(2, 2);
+
+console.log('addTwoNumbers Exercise', addTwoNumbersResult);
+
+function convertHoursToMinutes(hour) {
+  return hour * 60;
+}
+
+var convertHoursToMinutesResult = convertHoursToMinutes(2);
+
+console.log('convertHoursToMinutes Exercise', convertHoursToMinutesResult);
+
+function getGreeting(name) {
+  return '"Hello World!"';
+}
```

Comment:
for this function we'll want to concatenate our `name` parameter after the string `'Hello '`

```js
function getGreeting(name) {
  return 'Hello ' + name + '!';
}
```